Plotting: add alias `--backtest-filename` for `--export-filename`

TheJoeSchr opened the pull request:
## Summary
makes it easier to discover how to use this argument


This is just an initial approach. If it seems to be good, I can also port it over to plot-dataframe etc.

If there is a better way to do this, please let me know...

TheJoeSchr committed:
Plotting: add alias `--backtest-filename` for `--export-filename`

makes it easier to discover how to use this argument

xmatthias commented on freqtrade/commands/cli_options.py:
```diff
@@ -182,11 +182,17 @@ def __init__(self, *args, **kwargs):
+        metavar="PATH",
+    ),
+    "backtestfilename": Arg(
+        "--backtest-filename",
```
I'd clearly not use 2 different arguments / configuration keys for the same thing.
That's pretty dangerous, and is bound to fail assuming one comes from the configuration (then you got both variables available - and will wonder why the wrong file is loaded).

you can assign a 2nd argument to the same key eventually, and maybe change the help text to something like `Filename for backtesting results" - so it works for both loading and saving.

TheJoeSchr replied:
you mean like this? I haven't tested commit 298815d yet, I wanted to see if I understood you right

xmatthias replied:
Yes exactly

TheJoeSchr committed:
fix: don't use different configuration keys, just add as 2nd argument

TheJoeSchr committed:
fix: linter

TheJoeSchr commented:
I think this is ready now?

I did this:

- rebased on current `develop`
- fixed flake8 linter issue
- tested `--backtest-filename` w/ `plot-profit`
- tested `--export-filename` w/ `plot-dataframe`
- tested `--backtest-filename` w/ `plot-profit`
- tested `--export-filenames` w/ `plot-dataframe`

xmatthias reviewed (approved):
For sure 👍
